src/core/reactor.cc

```diff
@@ -95,7 +95,11 @@ module;
 #include <sys/mman.h>
 #include <sys/utsname.h>
 #include <linux/falloc.h>
+#ifndef SEASTAR_HAVE_SYSTEMTAP_SDT
```

Comment:
this needs to be #ifdef or it includes sys/sdt.h if not found

Reply:
@a16bitsysop ahh, right. fixed.